### DISCUSSION TITLE
App hanging for at least 2000 ms.

### DISCUSSION THREAD

**Discussion by MShokry** [getsentry/sentry-react-native#3361] [Q&A]
Hi everyone, 

I've been encountering the "App hanging for at least 2000 ms" issue multiple times while using Sentry. The app seems to be experiencing significant delays, and I'm having trouble pinpointing the root cause of the problem.

Environment

   @sentry/react-native: ^5.11.1
    React Native: 0.72.4
    Platform: iOS

Additional Information

I've checked the following without success:

    Reviewed the Sentry documentation for common causes of performance issues.
    Ensured that my dependencies are up to date.
    Checked network requests for potential bottlenecks.

Screenshots:

<img width="728" alt="Screenshot 2023-10-23 at 10 33 36 AM" src="https://github.com/getsentry/sentry-react-native/assets/1007641/d0d85e90-6916-4a4f-960a-4f67db60c529">

**Comment by phuocantd** [comment]
same here

**Comment by usmanthesuper** [comment]
us
